.github/workflows/tests.yml

```diff
@@ -0,0 +1,68 @@
+name: "Tests"
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  Test:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    strategy:
+      matrix:
+        node_version: ['lts/*', 'node']
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Install Node
+        uses: dcodeIO/setup-node-nvm@master
+        with:
+          node-version: ${{ matrix.node_version }}
+      - name: Install Dependencies
+        run: npm ci
+      - name: Run Unit Tests 👩🏽‍💻
```

Comment:
Should we run lint before unit tests instead of a separate job? If it fails lint then it could fail fast.

Reply:
I think it would be better to run in parallel. We could turn off the fail-fast so it will always run both tests on node and lts and lint. For that matter we could also run unit tests and spec tests in parallel too if we wanted.